process.c

```diff
@@ -3035,8 +3038,23 @@ NORETURN(static VALUE f_exec(int c, const VALUE *a, VALUE _));
  *
  *  \String argument +command_line+ is a command line to be passed to a shell;
  *  it must begin with a shell reserved word, begin with a special built-in,
- *  or contain meta characters.
- *  It may also contain arguments and options for that command.
+ *  or contain meta characters:
+ *
+ *    exec('date')                         # Built-in.
+ *    exec('if true; then echo "Foo"; fi') # Shell reserved word.
+ *    [Needs example.]                     # Contains meta character.
```

Comment:
I think an example for this could be `exec('echo $0')` (since `$` is a meta character). It outputs `sh` UNIX systems.

Reply:
`date` is not a built-in command on POSIX (but on cmd.exe).
`echo` is built-in, it may be confusing as an example of meta character.

Reply:
A newline character (`\n`) is a meta character on all shells AFAIK.